js/notifications.js

```diff
@@ -46,34 +46,65 @@ weechat.factory('notifications', ['$rootScope', '$log', 'models', 'settings', fu
                 });
             });
         } else {
-            var notification = new Notification(title, {
-                body: body,
-                icon: 'assets/img/favicon.png'
-            });
-
-            // Save notification, so we can close all outstanding ones when disconnecting
-            notification.id = notifications.length;
-            notifications.push(notification);
+            
+            //If it detects the Windows object, that is, running on a Hosted Web App container, use the UWP toasts
+            if (typeof Windows !== 'undefined' && 
```

Comment:
please make this an else if with the above condition, that would preserve the old code as-is without indentation changes which make code changes unnecessarily hard to trace